UnityGLTF/Assets/UnityGLTF/Tests/Assets/SkyboxMaterial.mat

```diff
@@ -76,7 +76,7 @@ Material:
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
     - _Tex:
-        m_Texture: {fileID: 8900000, guid: 89f974ac697f8c348897a2cc7e13a754, type: 2}
+        m_Texture: {fileID: 0}
```

Comment:
Lost a texture reference here. Intentional?

Reply:
That was an old test asset, now removed